torchci/rockset/inductor/__sql/compilers_benchmark_performance.sql

```diff
@@ -103,6 +105,14 @@ results AS (
       CAST(memory_bandwidth AS FLOAT),
       0.0
     ) AS memory_bandwidth,
+    IF(TRY_CAST(dynamo_peak_mem AS FLOAT) IS NOT NULL,
```

Comment:
IIUC this is a raw memory usage number. Do we want this as peak utilization (percentage of available memory) instead, since this is what is used for internal tracking? Will leave that up to whoever monitors this dashboard though